translations/fr.yaml

```diff
@@ -0,0 +1,247 @@
+    chrome-select-camera: <em>La vue de la caméra n'est pas la bonne?</em> - comme s'il s'agit de la caméra intégrée de votre portable plutôt que d'une webcam USB? Vous pouvez sélectionner la bonne caméra à chrome://settings/content/camera (copiez et collez ceci comme une URL). Rafraichissez ensuite cette page pour essayer à nouveau. 
+    chrome-further-help: Vous pouvez aussi voir <a href="https://support.google.com/chrome/answer/2693767?hl=en" target="_blank" rel="noopener noreferrer"> ces instructions de Chrome </a>.
+    firefox-instructions-header: Instructions pour la configuration de votre caméra pour Firefox
+    firefox-directions-prompt: La première fois que vous chargerez cette page vous devriez voir une invite comme celle qui se trouve ci-dessous. Sélectionnez la caméra et le micro que vous voulez utiliser, ensuite sélectionnez «Se souvenir de ce choix» et cliquez «Permettre».
+    firefox-directions-set-perms: Si vous n'avez pas vu cette invite, ou si vous avez premièrement cliqué «Ne pas permettre», vous pouvez modifier les paramètres en cliquant sur l'icône représentant une caméra et/ou un micro rayé(es) qui se trouve à côté de l'URL. Cliquez le "X" qui se trouve à côté de chaque paramètre et rafraichissez la page. 
+    additional-troubleshooting-header: Cela ne fonctionne tout simplement pas - que puis-je faire d'autre?
+    additional-troubleshooting-intro: Premièrement, nous tenons à vous remercier pour votre aide car vous êtes un citoyen scientifique dévoué! Nous sommes désolés que vous ayez des difficultés. 
+    camera-check: Assurez-vous d'être en mesure de voir la vidéo de votre webcam à l'extérieur de Lookit. (Par exemple, sur un Mac, essayez Photobooth.)
+    try-another-browser: L'approche la plus simple serait d'utiliser un navigateur différent si vous en avez un qui est installé. Par exemple, si vous utilisez Firefox, essayez avec Chrome, ou vice versa. D'autre part, mettez à jour le navigateur si vous n'avez pas la version la plus récente. Vous pouvez vérifier ceci en sélectionnant «Chrome» -> «À propos de Chrome» ou «Firefox» -> «À propos de Firefox» dans la barre des menus. (Ceci est souhaitable de toute façon pour vous assurer d'avoir les mises à jour de sécurité les plus récentes). 
+    chrome-incognito: Utilisez-vous Chrome? Assurez-vous de ne pas utiliser une fenêtre Incognito - vous seriez incapable de sauvegarder et d'utiliser les paramètres pendant la durée l'étude. 
+    additional-troubleshooting-contact: Si vous n'arrivez toujours pas à trouver, c'est de notre faute. S'il vous plaît, faites-le-nous savoir pour que nous puissions améliorer le site! Écrivez-nous à lookit-tech@mit.edu avec une brève description de ce qui s'est passé. Veuillez aussi inclure le navigateur web que vous utilisez (Firefox, Chrome, IE, Safari, etc.), les mesures vous avez essayées, et l'ID de cette session.

```

Comment:
Missing word in line 242 - correction:
additional-troubleshooting-contact: Si vous n'arrivez toujours pas à trouver, c'est de notre faute. S'il vous plaît, faites-le-nous savoir pour que nous puissions améliorer le site! Écrivez-nous à lookit-tech@mit.edu avec une brève description de ce qui s'est passé. Veuillez aussi inclure le navigateur web que vous utilisez (Firefox, Chrome, IE, Safari, etc.), les mesures que vous avez essayées, et l'ID de cette session.